pogobuf/pogobuf.client.js

```diff
@@ -15,7 +15,10 @@ const RequestType = POGOProtos.Networking.Requests.RequestType,
     Responses = POGOProtos.Networking.Responses;
 
 const INITIAL_ENDPOINT = 'https://pgorelease.nianticlabs.com/plfe/rpc';
-const DEFAULT_MAP_OBJECTS_DELAY = 5;
+const DEFAULT_MAP_OBJECTS_DELAY = 30;
+const DEFAULT_MAP_OBJECTS_DISTANCE = 10;
+const DEFAULT_FORT_INTERACTION_RANGE = 40;
+const DEFAULT_FORT_INTERACTION_RANGE_FAR = 1000;
```

Comment:
[Line 815 in fork `hunterjm:feature/distance`](https://github.com/hunterjm/pogobuf/blob/edb2cc47eed3be04e201b1fe4d4a824e3d269c14/pogobuf/pogobuf.client.js#L815).